Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorInstance.cpp

```diff
@@ -670,8 +685,15 @@ namespace AZ::Render
             meshDescriptor.m_isAlwaysDynamic = true;
             meshDescriptor.m_excludeFromReflectionCubeMaps = true;
 
+            AZ::Render::CustomMaterialMap customMaterials;
+            for (const auto& materialAssignment : materials)
+            {
+                customMaterials.emplace(
+                    AZ::Render::CustomMaterialId{ materialAssignment.first.m_lodIndex, materialAssignment.first.m_materialSlotStableId },
+                    AZ::Render::CustomMaterialInfo{ materialAssignment.second.m_materialInstance, materialAssignment.second.m_matModUvOverrides });
+            }
```

Comment:
nit: More duplication... maybe this material assignment map to custom materials code could be in one shared location.